defaults/main.yml

```
@@ -20,6 +20,8 @@ certbot_certs: []
 certbot_create_command: >-
   {{ certbot_script }} certonly --standalone --noninteractive --agree-tos
   --email {{ cert_item.email | default(certbot_admin_email) }}
+  {{ '--rsa-key-size' if certbot_rsa_key_size else '' }}
+  {{ certbot_rsa_key_size if certbot_rsa_key_size else '' }}
```

May want to try something like this instead, shorter, and on one line
```suggestion
{{ (certbot_rsa_key_size is defined) | ternary('--rsa-key-size ' ~ certbot_rsa_key_size, '')  }}
```

---

@microjoe ^